files/sidebars/webassemblysidebar.yaml

```
@@ -4,58 +4,132 @@ sidebar:
   - type: section
     link: /WebAssembly
     title: WebAssembly
-  - title: Guides
-    details: open
-    children:
-      - link: /WebAssembly/Guides/Concepts
-        title: WebAssembly_concepts
-      - link: /WebAssembly/Guides/C_to_Wasm
-        title: Compiling_to_Wasm
-      - link: /WebAssembly/Guides/Rust_to_Wasm
-        title: Compiling_Rust_to_Wasm
-      - link: /WebAssembly/Guides/Using_the_JavaScript_API
-        title: JavaScript_API
-      - link: /WebAssembly/Guides/Understanding_the_text_format
-        title: Text_format
-      - link: /WebAssembly/Guides/Text_format_to_Wasm
-        title: Text_format_to_Wasm
-      - link: /WebAssembly/Guides/Loading_and_running
-        title: Loading_and_running
-      - link: /WebAssembly/Guides/Exported_functions
-        title: Exported_functions
-      - link: /WebAssembly/Guides/JavaScript_builtins
-        title: JavaScript_builtins
-      - link: /WebAssembly/Guides/Imported_string_constants
-        title: Imported_string_constants
+  - type: section
+    link: /WebAssembly/Guides
+    title: Guides
+  - link: /WebAssembly/Guides/Concepts
+    title: WebAssembly_concepts
+  - link: /WebAssembly/Guides/C_to_Wasm
+    title: Compiling_to_Wasm
+  - link: /WebAssembly/Guides/Existing_C_to_Wasm
+    title: Existing_C_to_Wasm
+  - link: /WebAssembly/Guides/Rust_to_Wasm
+    title: Compiling_Rust_to_Wasm
+  - link: /WebAssembly/Guides/Using_the_JavaScript_API
+    title: JavaScript_API
+  - link: /WebAssembly/Guides/Understanding_the_text_format
+    title: Text_format
+  - link: /WebAssembly/Guides/Text_format_to_Wasm
+    title: Text_format_to_Wasm
+  - link: /WebAssembly/Guides/Loading_and_running
+    title: Loading_and_running
+  - link: /WebAssembly/Guides/Exported_functions
+    title: Exported_functions
+  - link: /WebAssembly/Guides/JavaScript_builtins
+    title: JavaScript_builtins
+  - link: /WebAssembly/Guides/Imported_string_constants
+    title: Imported_string_constants
+  - type: section
+    link: /WebAssembly/Reference
+    title: Reference
   - title: JavaScript_interface
     details: open
```

I think we should have this collapsed in the initial state
```suggestion
    details: closed
```